packages/@coorpacademy-components/src/molecule/card/favorite.css

```diff
@@ -47,48 +45,30 @@
   color: inherit;
 }
 
-.showToolTip {
-  visibility: hidden;
-  background-color: color(black a(80%));
-  color: #fff;
-  text-align: center;
-  border-radius: 3px;
-  z-index: 1;
-  opacity: 0;
-  transition: opacity 0.8s;
+.favoriteWrapper {
   position: absolute;
-  max-width: 104px;
-  right: 45px;
-  top: 8px;
+  width: 100%;
 }
 
-.showToolTip::after {
-  content: "";
-  position: absolute;
-  top: 8px;
-  left: 100%;
-  border: solid 5px ;
-  border-color: transparent transparent transparent color(black a(80%));
+.favoriteWrapper .favorite + [class*="__react_component_tooltip"]:not([class*="toolTipReact"]) {
+  position: sticky;
+  top: unset !important;
```

Comment:
🤔

Reply:
-> `!important` + selectors specificity to override css props used with !important inside the external tooltip lib that we use (no other way to override it)